src/components/Assets/AssetsList.tsx

```diff
@@ -460,15 +459,11 @@ const AssetsList = () => {
           </div>
         </>
       )}
-      {typeof facility === "undefined" && (
+      {facility === null && (
```

Comment:
`==` on null will check against null and undefined

```suggestion
      {facility == null && (
```